js/modules/modules.go

```diff
@@ -1,3 +1,4 @@
+// Package modules defines interfaces to be implemented by js modules to be imported by k6.
```

Comment:
Not really happy with the wording, but I am working on some PoC stuff and I keep getting linter errors and decided to just get this out of the way :grimacing: